CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/zustand/issues/new).
+
+For usage questions, prefer [starting a discussion](https://github.com/pmndrs/zustand/discussions/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Documentation
+
+- Fork the [Zustand repo](https://github.com/pmndrs/zustand/) into your own Github account
+- Locally, clone down your fork
+- Separately, clone the [Pmdrs-Website](https://github.com/pmndrs/website}
+  - this runs most of the doc websites under the pmndrs banner, including react three fiber and (eventually) zustand
```

Comment:
```suggestion
  - this runs most of the doc websites under the pmndrs banner, including React Three Fiber and Zustand
```

 - Removed eventually because we're pretty close to having the docs in a good place thanks to everyones help
 - Altered case for consistency